.circleci/config.yml

```diff
@@ -81,6 +81,30 @@ jobs:
       - run:
           name: Publish package
           command: "./node_modules/.bin/lerna publish --message 'chore: release new versions' --yes"
+  generate-repo:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys:
+          fingerprints:
+            - '95:74:63:db:82:da:b3:e6:57:0c:3a:bd:3f:ff:ce:c6'
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+      - run: git config --global user.email circleci@circleci
+      - run: git config --global user.name CircleCI
+      - run: 
+          name: Publish init output to GitHub
+          command: |
+            cd packages/create
+            git clone git@github.com:open-wc/init-output-test.git
+            yarn start --type scaffold --scaffoldType wc --features testing demoing building demoing --tagName init-output-test --scaffoldFilesFor testing demoing building --installDependencies false --writeToDisk true --typescript no --overwriteFile always
```

Comment:
This ensures we use the _latest_ generator at all times. Which is needed in time like now when we're adding CLI properties like `--overwriteFile`.